src/main/java/reborncore/common/powerSystem/TilePowerAcceptor.java

```diff
@@ -65,28 +59,29 @@
 	public double powerLastTick;
 	public boolean checkOverfill = true; //Set to flase to disable the overfill check.
 
-	List<ExternalPowerHandler> powerManagers;
+	private List<ExternalPowerHandler> powerManagers;
 
 	public TilePowerAcceptor() {
 		checkTeir();
-		final TilePowerAcceptor tile = this;
-		powerManagers = ExternalPowerSystems.externalPowerHandlerList.stream()
-			.map(externalPowerManager -> externalPowerManager.createPowerHandler(tile))
-			.collect(Collectors.toList());
+		setupManagers();
 	}
 
+	// don't manually set tiers
+	@Deprecated
 	public TilePowerAcceptor(EnumPowerTier tier) {
-		checkTeir();
-		setupManagers();
+		this();
 	}
 
-
 	private void setupManagers(){
 		final TilePowerAcceptor tile = this;
 		powerManagers = ExternalPowerSystems.externalPowerHandlerList.stream()
 			.map(externalPowerManager -> externalPowerManager.createPowerHandler(tile))
 			.filter(Objects::nonNull)
 			.collect(Collectors.toList());
+
+		if(RebornCoreConfig.enableFE) {
```

Comment:
Any reason this is hard coded here, and not registered with the API?

Example: 

https://github.com/TechReborn/TechReborn-ModCompatibility/blob/1.12.2/src/main/java/techreborn/compatmod/ic2/RebronCoreIC2.java#L13

EDIT: seen comments.